core-bundle/contao/themes/flexible/styles/main.css

```diff
@@ -1529,6 +1528,62 @@ table.tl_listing {
 	margin-left: 1px;
 }
 
+.operations > ul {
+	display: flex;
+	justify-content: flex-end;
+	align-items: center;
+	gap: 3px;
+	flex-wrap: wrap;
+	list-style: none;
+}
+
+.operations-menu-container {
+	position: relative;
+	align-items: center;
+}
+
+.operations-menu {
+	position: fixed;
+	z-index: 10;
+	display: flex;
+	flex-direction: column;
+	gap: 5px;
+	margin: 0;
+	padding: 10px !important;
+	text-align: left;
+	background: var(--content-bg) !important;
+	border: 1px solid var(--content-border);
+	list-style: none;
+	visibility: hidden;
+	pointer-events: none;
+}
+
+.operations-menu.show {
+	visibility: visible;
+	pointer-events: auto;
+}
+
+.operations-menu li {
+	display: block !important;
+}
+
+.operations-menu a {
+	display: flex;
+	gap: 5px;
+	align-items: center;
+	white-space: nowrap;
+}
+
+.operations-menu .virtual-title a:after {
+	content: attr(title);
+	padding-left: 5px;
+}
+
+.operations-menu .virtual-title a[data-original-title]:after {
+	content: attr(data-original-title);
+	padding-left: 5px;
+}
+
```

Comment:
```suggestion
}

.tl_content_right:has(.operations-menu.show) {
	z-index: 2;
}
```

What would we do without the `:has` selector :D

![image](https://github.com/user-attachments/assets/3ff39ece-d398-4ffe-b636-6a55e887f7e8)

Reply:
There is already a `z-index:10` on `.operations-menu`. I guess we need to `!important` that instead of just using a longer selector to override?

Reply:
Nope, same issue as always. The stacking context matters. The next sibling of `tl_right` has a the same `z-index`, thus higher stacking-context so `z-index: 1` of `tl_right` would completely nullify the `z-index: 10`.

TL;DR: It just has a z-index of 10 within it's parent that has a z-index of 1... not in the whole document :)

Reply:
that has become irrelevant with `position:fixed`, right?

Reply:
> that has become irrelevant with `position:fixed`, right?

Nope, you can test it in any content-element right now when opening the context-menu with the 3 dots:
![image](https://github.com/user-attachments/assets/d2e0ebf4-f02d-48b7-9c8b-22c48f7b689f)